packages/yoga/src/Banner/native/Banner.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+
+import styled from 'styled-components';
+import { borders, margins } from '@gympass/yoga-system';
+import {
+  func,
+  oneOf,
+  string,
+  checkPropTypes,
+  elementType,
+  shape,
+} from 'prop-types';
+
+import Box from '../../Box';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import Text from '../../Text';
+
+const StyledBanner = styled.View`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        components: { banner },
+        spacing,
+        colors: {
+          feedback: {
+            [variant]: backgroundColor = {
+              light: banner.defaultBackgroundColor,
+            },
+          },
+        },
+      },
+    },
+  }) => `
+    background-color: ${backgroundColor.light};
+    padding: ${spacing.xxsmall}px
+      ${spacing.small}px;
+    border-radius: ${banner.border.radius}px;
+  `}
+
+  ${margins}
+  
+  ${borders}
+`;
+
+const BannerActionButton = ({ button: { label, action }, ...props }) => (
+  <Box as={Button.Text} small secondary onPress={action} {...props}>
+    {label}
+  </Box>
+);
+
+BannerActionButton.propTypes = {
+  button: shape({
+    label: string.isRequired,
+    action: func.isRequired,
+  }).isRequired,
+};
+
+/** A banner is a component that displays a prominent message. It can have related action buttons on it or not. */
+const Banner = ({
+  icon,
+  variant,
+  message,
+  primaryButton,
+  secondaryButton,
+  ...props
+}) => (
+  <StyledBanner variant={variant} {...props}>
+    <Box flexDirection="row" flex={1} alignItems="center">
+      {icon && (
+        <Icon as={icon} size="medium" fill="secondary" marginRight="xxsmall" />
+      )}
+      <Text.Small flex={1} marginVertical="xxsmall">
+        {message}
+      </Text.Small>
+      {!!primaryButton && !secondaryButton && (
+        <BannerActionButton button={primaryButton} marginLeft="xxsmall" />
+      )}
+    </Box>
+    {primaryButton && secondaryButton && (
```

Comment:
why the first one (80) it's `!!primaryButton` and the second (84) `primaryButton`?

Reply:
For any specific reason, I'm going to update it to use the same pattern at both comparison

Reply:
Done!